src/olympia/reviewers/utils.py

```diff
@@ -467,7 +467,6 @@ def get_actions(self, request):
                          'versions. The comments will be sent to the '
                          'developer.'),
             'available': (
-                self.addon.type != amo.ADDON_STATICTHEME and
```

Comment:
Rejecting multiple versions kinda doesn't make sense for a theme, since they should only have a single version. Could you make it work with the `reject` action instead ?

Reply:
Sure. I'll modify it and submit it.

Reply:
fwiw, themes _can_ have multiple versions, but it's rare in reality.

Reply:
Ah, in that case, maybe we should remove that restriction and let the `reject_multiple_versions` be available for theme reviewers as well.

Reply:
Yes, I understand.